run_benchmarks.py

```diff
@@ -0,0 +1,279 @@
+import os
+from subprocess import Popen, PIPE, STDOUT
+import argparse
+import re
+import signal
+import sys
+import logging
+from glob import glob
+import shutil
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(levelname)s - %(message)s",
+)
+
+BASE_DIR = "build/Release"
+
+benchmarks = {
+    "coro_fibs": ["coro-fibs/coro-fibs"],
+    "excret": ["excret/exc_ret", "excret/exc_ret_noexc"],
+    "inline": ["inline/extswap"],
+    "noexcept_qsort": ["noexcept-qsort/exc_partition", "noexcept-qsort/exc_qsort"],
+    "ranges_filter": ["ranges-filter/filter"],
+    "ranges_projector": ["ranges-projector/projector"],
+    "virtual_inherit": ["virtual-inherit/virtinh"],
+    "virtual_inline": ["virtual-inline/virtinl"],
+    "virtual_overhead": [
+        "virtual-overhead/virtual",
+        "virtual-overhead/virtual-shuffle",
+    ],
+}
+
+
+def clean_ansi_escape_sequences(text):
+    """
+    Removes ANSI escape sequences from the given text.
+
+    Args:
+        text (str): The text with potential ANSI escape sequences.
+
+    Returns:
+        str: Cleaned text without ANSI escape sequences.
+    """
+    ansi_escape = re.compile(r"\x1B[@-_][0-?]*[ -/]*[@-~]")
+    return ansi_escape.sub("", text)
+
+
+def help_msg():
```

Comment:
Please make use of `argparse`s native help message instead of writing it yourself